source/includes/harvest/_users.md

```diff
@@ -143,11 +143,11 @@ id | The ID of the user whose job permissions to retrieve
 
 An array of objects of the following:
 
-Parameter | Description
```

Comment:
Unrelated add/remove job permissions. I added a Type column to keep it consistent.